controller/src/main/java/org/jboss/as/controller/resource/InterfaceDefinition.java

```diff
@@ -342,6 +342,11 @@ protected void addOperationParameterValueTypeDescription(ModelNode node, String
             private void setValueType(ModelNode node) {
                 node.get(ModelDescriptionConstants.VALUE_TYPE).set(ModelType.STRING);
             }
+
+            @Override
+            public AttributeDefinition getValueType() {
```

Comment:
This is fishy... the setter accepts a `ModeNode` while the getter returns an `AttributeDefinition`?

Reply:
I agree but the setter is private :)

Reply:
The visibility of the gettter/setter do not matter. I'd expect to get consistent types for the getter and the setter.